backtrace-library/build.gradle

```diff
@@ -5,11 +5,18 @@ android {
     compileSdkVersion 28
 
     defaultConfig {
-        minSdkVersion 19
+        minSdkVersion 21
```

Comment:
If you want to change the minimum SDK, please also update this information in README.md

Reply:
done - also one thing to note - we will take care of commercialization notes later.